.github/workflows/test-unreleased.yml

```diff
@@ -53,14 +87,32 @@ jobs:
           ruby -v
           rake bundle:clean
           rake bundle:install
+        working-directory: ruby-client
 
       - name: opensearch-ruby
-        run: cd opensearch-ruby && bundle exec rake test:all
+        working-directory: ruby-client/opensearch-ruby
+        run: bundle exec rake test:all
+
       - name: opensearch-transport
-        run: cd opensearch-transport && bundle exec rake test:all
+        working-directory: ruby-client/opensearch-transport
+        run: bundle exec rake test:all
+
       - name: opensearch-api
-        run: cd opensearch-api && bundle exec rake test:spec
+        working-directory: ruby-client/opensearch-api
+        run: bundle exec rake test:spec
```

Comment:
For another time, we should probably make this `test:all` as well.

Reply:
`test:all` will also include unit tests with mocked transport layer, which don't hit the OS cluster that we setup for each Matrix entry. Wouldn't that be redundant for this workflow where we focus on integration tests of different unreleased versions of OS?

Reply:
It would. I still think it's mental overhead, maybe there's a better solution. Either way it's NBD.